### PR TITLE
Cleaning up some logic in  tools/shared/cwrap_common.py

### PR DESCRIPTION
**Pull request by joesho112358**
Noticed some code that needed some adjustment

### CODE REVIEW

File: tools/shared/cwrap_common.py [hunk 1]
```diff
@@ -47,19 +47,23 @@ def set_declaration_defaults(declaration: Declaration) -> None:
         )
     else:
         declaration["type_wrapper_name"] = declaration["name"]
-    # TODO: Uggggh, parsing the schema string here, really???
```
Comment (ezyang) [inline tools/shared/cwrap_common.py:50]: the comment still applies lol

**Comment by ezyang** [comment]
I think this logic is entirely dead. Can you try deleting this file and the imports from shared/__init__.py? thanks

**Comment by joesho112358** [comment]
will look into it

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**